quantization/nlp/bert/trt/e2e_tensorrt_bert_example.py

```diff
@@ -207,19 +207,30 @@ def inference(data_reader, ort_session):
     vocab_file = "./squad/vocab.txt"
     augmented_model_path = "./augmented_model.onnx"
     qdq_model_path = "./qdq_model.onnx"
-    sequence_lengths = [384]
-    calib_num = 100
-    op_types_to_quantize = ['MatMul', 'Transpose', 'Add']
+    sequence_lengths = [128]
+    calib_num = 10
```

Comment:
Is 10 enough to get good calibration results?

Reply:
10 does seem too few.

Reply:
Yes, it's a typo. Made it back to 100.